src/index.js

```diff
@@ -136,6 +140,30 @@ class MongooseStore {
             this.result(fn, e);
         }
     }
+
+    keys(fn) {
+        try {
+            return this.model
+                .find({})
+                .then(records => {
+                    if (!records) {
```

Comment:
I don't think this condition is necessary. If there are no records, then an empty array is returned which is treated as true. And if there was an error - we'll run into catch. So i think it's impossible that we get into this condition ever.